### ISSUE TITLE
Issue updating module

### ISSUE THREAD

**Issue by JaspervRijbroek** [npm/npm#8252]
Hello I am trying to update my published module to a new version.
When I bumped the version I got a successful message.

This however was to the wrong registry (private one or so I thought) so I did a ```npm unpublish``` with the latest version number.

After this I tried to push to the normal repository but I got an error 403.
I followed som issues and executed the command ```npm view raddish time``` and I saw the latest version but when I execute the command ```npm view raddish versions``` I don't see the latest version (being 2.4.8).

Could you help me with this issue?
```
This is my log file:
0 info it worked if it ends with ok
1 verbose cli [ 'node', '/usr/local/bin/npm', 'publish' ]
2 info using npm@2.9.1
3 info using node@v0.12.2
4 verbose publish [ '.' ]
5 silly cache add args [ '.', null ]
6 verbose cache add spec .
7 silly cache add parsed spec { raw: '.',
7 silly cache add   scope: null,
7 silly cache add   name: null,
7 silly cache add   rawSpec: '.',
7 silly cache add   spec: '/Users/Jasper/Projects/Personal/Raddish',
7 silly cache add   type: 'directory' }
8 verbose addLocalDirectory /Users/Jasper/.npm/raddish/2.4.8/package.tgz not in flight; packing
9 verbose tar pack [ '/Users/Jasper/.npm/raddish/2.4.8/package.tgz',
9 verbose tar pack   '/Users/Jasper/Projects/Personal/Raddish' ]
10 verbose tarball /Users/Jasper/.npm/raddish/2.4.8/package.tgz
11 verbose folder /Users/Jasper/Projects/Personal/Raddish
12 info prepublish raddish@2.4.8
13 verbose addLocalTarball adding from inside cache /Users/Jasper/.npm/raddish/2.4.8/package.tgz
14 silly cache afterAdd raddish@2.4.8
15 verbose afterAdd /Users/Jasper/.npm/raddish/2.4.8/package/package.json not in flight; writing
16 verbose afterAdd /Users/Jasper/.npm/raddish/2.4.8/package/package.json written
17 silly publish { name: 'raddish',
17 silly publish   description: 'Another MVC framework.',
17 silly publish   version: '2.4.8',
17 silly publish   author:
17 silly publish    { name: 'Jasper van Rijbroek',
17 silly publish      email: 'jasper@jvar.nl',
17 silly publish      url: 'http://jvar.nl' },
17 silly publish   homepage: 'http://getraddish.com',
17 silly publish   main: './index.js',
17 silly publish   repository:
17 silly publish    { type: 'git',
17 silly publish      url: 'git+https://github.com/JaspervRijbroek/raddish.git' },
17 silly publish   scripts:
17 silly publish    { test: 'mocha tests/* -R spec -t 15000',
17 silly publish      'test-travis': 'istanbul cover node_modules/mocha/bin/_mocha --report lcovonly -- --reporter spec --check-leaks -t 15000 tests/' },
17 silly publish   dependencies:
17 silly publish    { mysql: '2.6.2',
17 silly publish      formidable: '1.0.17',
17 silly publish      bluebird: '2.9.25',
17 silly publish      mime: '1.3.4',
17 silly publish      usage: '0.7.0',
17 silly publish      'update-notifier': '0.4.0',
17 silly publish      'socket.io': '1.3.5',
17 silly publish      clone: '1.0.2',
17 silly publish      mongodb: '2.0.28',
17 silly publish      sqlite3: '3.0.6',
17 silly publish      'universal-query': '*' },
17 silly publish   devDependencies:
17 silly publish    { mocha: '2.2.4',
17 silly publish      should: '6.0.1',
17 silly publish      istanbul: '0.3.13',
17 silly publish      'mock-req': '0.2.0',
17 silly publish      'mock-res': '0.2.1' },
17 silly publish   keywords:
17 silly publish    [ 'RAD',
17 silly publish      'framework',
17 silly publish      'promises',
17 silly publish      'MVC',
17 silly publish      'DCI',
17 silly publish      'fallback',
17 silly publish      'multidatabase',
17 silly publish      'streams' ],
17 silly publish   readme: 'Raddish Framework\n=================\n\n[![Raddish Logo](http://getraddish.com/assets/images/logo.svg)](http://getraddish.com)\n\n[![Build Status](https://api.travis-ci.org/JaspervRijbroek/raddish.svg)](https://travis-ci.org/JaspervRijbroek/raddish)\n[![Code Climate](https://codeclimate.com/github/JaspervRijbroek/raddish.png)](https://codeclimate.com/github/JaspervRijbroek/raddish)\n[![Coverage Status](https://coveralls.io/repos/JaspervRijbroek/raddish/badge.png?branch=develop)](https://coveralls.io/r/JaspervRijbroek/raddish?branch=develop)\n[![Inline docs](http://inch-ci.org/github/JaspervRijbroek/raddish.svg?branch=develop&style=shields)](http://inch-ci.org/github/JaspervRijbroek/raddish)\n\nWhat is Raddish Framework\n-------------------------\n\nRaddish is an open-source NodeJS Framework for web applications for modern web applications. \nDeveloped and maintained by passionate developers.\n\nThe design pattern based architecure gives more flexibility to your applications.\nBasically everything being written becomes resuable and extensible pieces of code.\n\nRaddish uses a [component based architecure](http://en.wikipedia.org/wiki/Component-based_software_engineering) seperated by applications,\nworking together accoring the [Hierarchical Model-View-Contoller](http://en.wikipedia.org/wiki/Hierarchical_model%E2%80%93view%E2%80%93controller) **(HMVC)** pattern.\n\nWhy Raddish Framework\n---------------------\n\nRaddish is build to help developers create a fast and reliable backend for web applications,\nRaddish tries to solve this by doing to following\n\n- Provide a solid, modern and lean architecture for building applications\n- Allow components to be flexible, reusable and extendable with ease.\n\nContributing\n------------\n\nWe love you contributing to Raddish Framework, this can be anything from bug fixes to features and inprovements.\nWe do appreciate some guidelines in contributes.\n\nWe use the [GitFlow](http://nvie.com/posts/a-successful-git-branching-model/) branching model which keeps a nice separation of bugfixes and features.\n\nWhere to send your pull request to:\n\n- When sending a pull request for a bugfix please prepend ```bugfix/``` to your branch name and send it to the ```develop``` branch so the bugfix will be added to master and develop. (non-breaking changes)\n- If you are sending a pull request for a feature please prepend ```feature/``` to your branch name and send it to the ```develop``` branch, this to ensure the feature to be added to the upcoming release. (minor/ breaking code changes).\n- For other pull requests please send them directly to the ```develop``` branch.',
17 silly publish   readmeFilename: 'README.md',
17 silly publish   gitHead: 'ed72a94eda900bf2316aadbbeace53523a2e9318',
17 silly publish   bugs: { url: 'https://github.com/JaspervRijbroek/raddish/issues' },
17 silly publish   _id: 'raddish@2.4.8',
17 silly publish   _shasum: '43c24aeedec588d09633a567f7eeaa18b87dc752',
17 silly publish   _from: '.' }
18 verbose getPublishConfig undefined
19 silly mapToRegistry name raddish
20 silly mapToRegistry using default registry
21 silly mapToRegistry registry http://registry.npmjs.org/
22 silly mapToRegistry uri http://registry.npmjs.org/raddish
23 verbose publish registryBase http://registry.npmjs.org/
24 silly publish uploading /Users/Jasper/.npm/raddish/2.4.8/package.tgz
25 verbose request uri http://registry.npmjs.org/raddish
26 verbose request always-auth set; sending authorization
27 info attempt registry request try #1 at 12:06:10 PM
28 verbose request id bea98535c4a3e4c2
29 http request PUT http://registry.npmjs.org/raddish
30 http 403 http://registry.npmjs.org/raddish
31 verbose headers { 'content-type': 'application/json',
31 verbose headers   'cache-control': 'max-age=60',
31 verbose headers   'content-length': '95',
31 verbose headers   'accept-ranges': 'bytes',
31 verbose headers   date: 'Wed, 13 May 2015 10:06:11 GMT',
31 verbose headers   via: '1.1 varnish',
31 verbose headers   connection: 'keep-alive',
31 verbose headers   'x-served-by': 'cache-ams4135-AMS',
31 verbose headers   'x-cache': 'MISS',
31 verbose headers   'x-cache-hits': '0',
31 verbose headers   'x-timer': 'S1431511570.316841,VS0,VE986' }
32 verbose request invalidating /Users/Jasper/.npm/registry.npmjs.org/raddish on PUT
33 error publish Failed PUT 403
34 verbose stack Error: "You cannot publish over the previously published version 2.4.8." : raddish
34 verbose stack     at CachingRegistryClient.<anonymous> (/usr/local/lib/node_modules/npm/node_modules/npm-registry-client/lib/request.js:247:14)
34 verbose stack     at Request._callback (/usr/local/lib/node_modules/npm/node_modules/npm-registry-client/lib/request.js:170:14)
34 verbose stack     at Request.self.callback (/usr/local/lib/node_modules/npm/node_modules/request/request.js:368:22)
34 verbose stack     at Request.emit (events.js:110:17)
34 verbose stack     at Request.<anonymous> (/usr/local/lib/node_modules/npm/node_modules/request/request.js:1219:14)
34 verbose stack     at Request.emit (events.js:129:20)
34 verbose stack     at IncomingMessage.<anonymous> (/usr/local/lib/node_modules/npm/node_modules/request/request.js:1167:12)
34 verbose stack     at IncomingMessage.emit (events.js:129:20)
34 verbose stack     at _stream_readable.js:908:16
34 verbose stack     at process._tickCallback (node.js:355:11)
35 verbose statusCode 403
36 verbose pkgid raddish
37 verbose cwd /Users/Jasper/Projects/Personal/Raddish
38 error Darwin 14.3.0
39 error argv "node" "/usr/local/bin/npm" "publish"
40 error node v0.12.2
41 error npm  v2.9.1
42 error code E403
43 error "You cannot publish over the previously published version 2.4.8." : raddish
44 error If you need help, you may report this error at:
44 error     <https://github.com/npm/npm/issues>
45 verbose exit [ 1, true ]
```

**Comment by KenanY** [comment]
```
43 error "You cannot publish over the previously published version 2.4.8." : raddish
```

@JaspervRijbroek You cannot publish `2.4.8` because it already exists; the `npm view raddish time` should show you that. It will not appear in `npm view raddish versions` because it has been unpublished. Simply bumping the version number should fix this.

**Comment by othiym23** [comment]
@KenanY is correct – if you run `npm version patch && npm publish`, everything should work out. Closing as resolved.
